main.go

```diff
@@ -218,16 +218,13 @@ func logError(err error, response *jira.Response) {
 func junit2csv(testSuites []junit.Suite, p params, output io.Writer) error {
 	w := csv.NewWriter(output)
 	header := []string{
+		"BuildId",
 		"Timestamp",
 		"Classname",
 		"Name",
 		"Duration",
 		"Status",
 		"JobName",
-		"Orchestrator",
-		"BaseLink",
-		"BuildLink",
-		"BuildTag",
```

Comment:
I think build tag could be useful to see all tests for given commit this will include retries